From 02836661c83fc378542de81c13738edca508f5eb Mon Sep 17 00:00:00 2001
From: Daniel Abrecht <public@danielabrecht.ch>
Date: Sun, 19 Dec 2021 13:58:31 +0100
Subject: [PATCH] arm64: dts: rk3399-pinephone-pro: fix modem enable gpio

The eg25 W_DISABLE pin enables airplane mode when low, which disables RF.
So high -> RF enabled, low -> RF disabled.

Signed-off-by: Daniel Abrecht <public@danielabrecht.ch>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index f4073d9a6fca..ef18cac88ac5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -1060,7 +1060,7 @@ modem {
 		power-supply = <&vcc_4g>;
 		vbus-supply = <&vcc_4g_5v>;
 
-		enable-gpios = <&gpio0 RK_PB0 GPIO_ACTIVE_LOW>; // W_DISABLE#
+		enable-gpios = <&gpio0 RK_PB0 GPIO_ACTIVE_HIGH>; // W_DISABLE#
 		reset-gpios = <&gpio3 RK_PB0 GPIO_ACTIVE_HIGH>;
 		status-gpios = <&gpio3 RK_PA6 GPIO_ACTIVE_HIGH>;
 		pwrkey-gpios = <&gpio0 RK_PB5 GPIO_ACTIVE_HIGH>;
-- 
2.30.2

